packages/examples/src/fixie-corpus.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * This is a simple demo showing how to query a corpus of documents through the Fixie service.
+ * The app only sends a single query to a corpus, and dumps out the raw text of the top 4 chunks.
+ */
+
+import { ScoredChunk, FixieCorpus } from 'ai-jsx/batteries/docs';
+import { showInspector } from 'ai-jsx/core/inspector';
+
+// This is the corpus ID for the docs.ai-jsx.com corpus.
+const FIXIE_CORPUS_ID = '07d42038-0c0c-4108-8b9e-163e0a4ce683';
+
+function ChunkFormatter({ doc }: { doc: ScoredChunk<any> }) {
+  return (
+    <>
+      {'\n\n'}Chunk from source: {doc.chunk.metadata?.source}
```

Comment:
Won't this always be null right now? Might make sense to omit this line until we're providing something useful here.